_html/templates/register-form-page.php

```diff
@@ -0,0 +1,643 @@
+<?php 
+  /** 
+  * Sponsors template.
+  */
+
+  include "inc/header.inc.php";
+  include "modules/header.php";
+?>
+
+<div>
```

Comment:
Why isn't all this mark-up in a different php file? Like it was done for all other templates? (a separate 'module')